public/index.tsx

```diff
@@ -26,6 +26,7 @@ const routes = [
   {
     path: "viewer",
     element: <AppWrapper />,
+    errorElement: <ErrorPage />,
```

Comment:
Added the ErrorPage to help with error handling in the App. This will catch any errors that aren't directly handled by the image viewer component.